src/main/java/life/qbic/subscriptions/subscriptions/CancellationRequest.java

```diff
@@ -0,0 +1,10 @@
+package life.qbic.subscriptions.subscriptions;
+
+/**
+ * <b>Record CancellationRequest</b>
+ *
+ * <p>Holds information about the user id and project, a person with a given user id
+ * should be removed from project notification subscriptions.</p>
+ */
+public record CancellationRequest(String project, String userId) {}
```

Comment:
What is a record?

Reply:
https://docs.oracle.com/en/java/javase/14/language/records.html